### DISCUSSION TITLE
Add documentation how to use fuzzers

### DISCUSSION THREAD

**Discussion by nlohmann** [nlohmann/json#3477] [Ideas]
From https://github.com/nlohmann/json/issues/3475#issuecomment-1120359121:

- Add documentation how to use AFL.
- Add documentation how to use LibFuzzer.
- Add documentation how OSS-Fuzz is relying on this repo.

**Comment by falbrechtskirchinger** [comment]
```
find tests/data -size -5k -name *.bjdata | xargs -I{} cp "{}" fuzz-testing/testcases
find: ‘tests/data’: No such file or directory
```

Remember to document the test data location.

**Comment by falbrechtskirchinger** [comment]
Also:
> Note: AFL hasn't been updated for a couple of years; while it should still work fine, a more complex fork with a variety of improvements and additional features, known as AFL++, is available from other members of the community and might be worth checking out.

https://github.com/AFLplusplus/AFLplusplus

**Reply by nlohmann** [reply]
I think that was the reason I could not get it running quickly on macOS. So LibFuzzer it is. I'll check it out later today.

**Comment by nlohmann** [comment]
I added a PR: https://github.com/nlohmann/json/pull/3478

**Reply by nlohmann** [reply]
Done.
